### DISCUSSION TITLE
请问 umi 入口文件怎么修改

### DISCUSSION THREAD

**Discussion by ghost** [umijs/umi#6485] [Q&A]
当前业务场景是做主应用页面有多个子应用（umi 开发的），子应用的根节点 id 会有冲突，所以会有冲突，需要修改 umi  入口文件，请问 umi 入口文件怎么修改

![image](https://user-images.githubusercontent.com/6204303/116054691-82911a80-a6ae-11eb-8a80-40048c2656fc.png)

**Comment by phobal** [comment]
https://umijs.org/zh-CN/config#mountelementid
